bridgesync/bridgesync.go

```diff
@@ -326,13 +326,21 @@ func (s *BridgeSync) GetLastProcessedBlock(ctx context.Context) (uint64, error)
 	return s.processor.GetLastProcessedBlock(ctx)
 }
 
-func (s *BridgeSync) GetBridgeRootByHash(ctx context.Context, root common.Hash) (*tree.Root, error) {
+func (s *BridgeSync) GetExitRootByHash(ctx context.Context, root common.Hash) (*tree.Root, error) {
 	if s.processor.isHalted() {
 		return nil, sync.ErrInconsistentState
 	}
 	return s.processor.exitTree.GetRootByHash(ctx, root)
 }
 
+func (s *BridgeSync) GetClaimByGlobalIndex(ctx context.Context, globalIndex *big.Int) (Claim, error) {
+	if s.processor.isHalted() {
+		s.processor.log.Error("processor is halted, cannot get claim by global index")
```

Comment:
Suggesting to remove this, because of consistency. WDYT?

```suggestion
```

Reply:
Fix: https://github.com/agglayer/aggkit/pull/838/commits/a0a491d0e1a9299e71c55ce098802a68184555fe